shared/src/androidMain/kotlin/io/customer/shared/util/PlatformUtil.kt

```diff
@@ -0,0 +1,9 @@
+package io.customer.shared.util
+
+import java.util.*
+
+internal actual fun getPlatformUtil(): PlatformUtil = AndroidPlatformUtil()
```

Comment:
I agree, but I couldn't think of better name right now. I tried using `DatabaseUtil` kind of thing as we are only using this class to generate UUIDs for database, but that still didn't feel very good. Please let me know if you have something to suggest so I can update it.

Reply:
Honestly, I didn't have a name to suggest to you while making my initial code review because I didn't have a suggestion for a better name, haha! 

I also thought of the `DatabaseUtil` idea but felt that it might only have 1 function in it for a while. 

Maybe we don't need a util at all? I see a need for having a util if we need to mock the getUUID() function but I am not sure if we need to do that for this 1 function.

Reply:
Yes, exactly, I did it mainly for tests. I'll then rename it to `DatabaseUtil` for now, and we can refactor it later when needed. If it remains only one function till end and is not needed for tests, we can move it out of util. But let's hold this until we find better place for the method.